backend/src/main/java/planit/people/preparation/APIs/API_User.java

```diff
@@ -35,7 +35,7 @@ public ResponseEntity<UserCreationResponse> createNewUser(@RequestBody DTO_Code
         }
     }
 
-    @GetMapping(path = "getAllEmails/{planit-user-id}")
+    @GetMapping(path = "emails/{planit-user-id}")
```

Comment:
Maybe `planit-user/{planit-user-id}/emails`?

Reply:
I would leave it as it is, we are following the standard of adding parameters at the end of the path, I would leave it like this for consistency